Checklist/index.jsx

```diff
@@ -29,10 +31,21 @@ function ChecklistMain ({ chromeless, className, children, customize, style, sty
     }}
     className={classNames(baseClass, { chromeless }, className)}
     {...props}>
+    {title && renderTitle(title, classNames(classes.title))}
     {children}
   </ul>
 }
 
+function renderTitle (title, className) {
+  return (
+    <li>
+      <Paragraph.Primary className={className}>
+        <strong>{title}</strong>
```

Comment:
We are rejecting this one now since the `Block.Installments` component already behaves like it was intended.